app/controllers/api/v2/searches_controller.rb

```diff
@@ -96,6 +96,8 @@ def search_params
                                          :api_key,
                                          :audience,
                                          :content_type,
+                                         :created_since,
```

Comment:
I'm wondering if `:created_after` and `:created_before` are better names here?

Reply:
@ddzz I'll be the first to admit the naming conventions here need some work (see: SRCH-3924 in the backlog).  I was trying to find a balance between not heavily reworking what is already in the codebase (`since`, `until`, `since_date`, and `until_date`), and still providing parameter names that make it clear what field is being searched.

Ultimately these params are become `created_since_date`, `created_until_date`, `since_date`, and `until_date` (in `lib/api/i14y_search_options.rb`) where they are then passed on to `app/models/filterable_search.rb`.  The FilterableSearch class is all through the models and controllers and keeping all the various logic there for the time being helps ensure that web searching and api searching functionality is the same (at least to the extent that various search options are allowed by the results api and/or made apparent in the web search ui for the various search types).

It's a tangle.  The other consideration is - at least with the minimal validation I've implemented to this point - if an API user hits a validation error in their query at the time being it's going to throw the name of that attribute as defined in `lib/api/i14y_search_options.rb` and passed to `app/models/filterable_search.rb` (so `created_since_date` and `created_until_date` for `created_since` and `created_until` or `since_date` and `until_date` for `updated_since` and `updated_until`).

Reply:
Thanks for the explanation. Since it does look like a tangle, I'm fine with keeping it as-is for now -- hopefully SRCH-3924 will improve the situation.